<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelRegistrationDialog</class>
 <widget class="QDialog" name="ModelRegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register Model Version</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout_">
   <item>
    <layout class="QFormLayout" name="form_layout_">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label_">
       <property name="text">
        <string>Model Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit_"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="path_label_">
       <property name="text">
        <string>Model Path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="path_layout_">
       <item>
        <widget class="QLineEdit" name="path_edit_">
         <property name="placeholderText">
          <string>models/best.pt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_button_">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="count_label_">
       <property name="text">
        <string>Training Images:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="count_edit_">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="notes_label_">
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="notes_edit_">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Optional notes about this model (e.g., hyperparameters, performance, purpose)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout_">
     <item>
      <spacer name="button_spacer_">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button_">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button_">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button_</sender>
   <signal>clicked()</signal>
   <receiver>ModelRegistrationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button_</sender>
   <signal>clicked()</signal>
   <receiver>ModelRegistrationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
